src/store/modules/accounts/hdWallet.js

```diff
@@ -94,31 +94,31 @@ export default {
       txBase64,
       opt: { modal = true, host = null },
     }) {
-      const sdk = rootGetters['sdkPlugin/sdk'];
+      const networkId = await (rootGetters['sdkPlugin/sdk'] ?? rootGetters['sdk13Plugin/sdk'].api).getNetworkId();
```

Comment:
This code is repeated [here](https://github.com/aeternity/superhero-wallet/pull/1940/files#diff-37085670e4cc4d0853535bb598e3c36b31ffb49af1badbebe2018c1591d51b9cR112) - maybe it could be extracted to a function or stored above?